cmd/libvirt-actuator/README.md

```diff
@@ -2,28 +2,114 @@
 
 The command allows to directly interact with the libvirt actuator.
 
+## How to prepare environment
+
+For running the libvirt actuator with Fedora 28 Cloud images.
+
+1. Provision instance in `packet.net`:
+   ```sh
+   $ ./hack/packet-provision.sh install
+   ```
+   and get the IP address (assuming it is `147.75.96.139`).
+
+1. Update ``~/.ssh/config`` to contain:
```

Comment:
So I can just type `ssh libvirtactuator` and log in without pointing to the generated ssh key and/or confirming I really want to connect.

Reply:
Wouldn't it be better to provide such command in README instead of constantly changing `.ssh/config` file?
It could be specified as "do this or that, both work".

Reply:
I think for these transient machines it is better to have something like:

`.ssh/config`:
```
Include conf.d/libvirtactuator/*.conf
```
and then you can drop files in there, but also equally blow them all away when not needed. It's easier to manage these through filesystem actions vis-a-vis editing `.ssh/config`.

Reply:
The only reason why I decided to update `.ssh/config` is version limitation:  https://superuser.com/a/1142813

I still have `openssh-clients-7.2p2-14.fc24.x86_64` and it's not trivial to bump to higher :-|

Reply:
**fc24** :open_mouth: You know that fc29 should land this month? :stuck_out_tongue:

Reply:
> **fc24**  You know that fc29 should land this month?

I couldn't wait. Went to f29 yesterday... :)